src/snowflake/connector/version.py

```diff
@@ -1,3 +1,3 @@
 # Update this for the versions
 # Don't change the forth version number from None
-VERSION = (3, 0, 3, None)
+VERSION = (3, 0, 4, None)
```

Comment:
Let's not change this until the next release?

Reply:
reverted